compatibility/bazel_tools/daml_trigger/daml_trigger.bzl

```diff
@@ -124,6 +124,21 @@ def daml_trigger_test(compiler_version, runner_version):
         compiler_version = version_to_name(compiler_version),
         runner_version = version_to_name(runner_version),
     )
+
+    # 1.16.0 is the first SDK version that uses LF 1.14, which is the earliest version that canton supports
+    use_canton = versions.is_at_least("2.0.0", runner_version) and versions.is_at_least("1.16.0", compiler_version)
+    use_sandbox_on_x = versions.is_at_least("2.0.0", runner_version) and not use_canton
+    if use_sandbox_on_x:
+        server = "@daml-sdk-{version}//:sandbox-on-x".format(version = runner_version)
+        server_args = ["--participant", "participant-id=sandbox,port=6865"]
```

Comment:
I’m not entirely sure if those tests are marked exclusive. If they are not, we cannot hardcode the port number. Marking them exclusive seems fine as a short term (up to 2.0) fix.

Reply:
Haven’t checked if the same applies to the daml script tests.

Reply:
They are tagged exclusive, as are the daml-script tests.

Reply:
perfect thanks for double checking